src/lib/service/catalog/suggestedLearning/suggestedLearningService.ts

```diff
@@ -0,0 +1,143 @@
+import { Course, User } from 'lib/model'
+import * as client from 'lib/service/catalog/courseCatalogueClient'
+
+import { getFullRecord } from '../../learnerRecordAPI/courseRecord/client'
+import { GetCoursesParams } from '../models/getCoursesParams'
+import { Suggestion } from './suggestion'
+import { SuggestionsMap } from './suggestionMap'
+import { SuggestionSection } from './suggestionSection'
+
+const DEFAULT_RECORDS_TO_SCAN_IN_ELASTIC = 200
+const RECORD_COUNT_TO_DISPLAY = 6
+const I_DONT_KNOW_AOW = "I don't know"
```

Comment:
Might be better to be more explicit about the types here (eg `const DEFAULT_RECORDS_TO_SCAN_IN_ELASTIC: number = 200`, for better debugging.